### ISSUE TITLE
[Feature Request]: Notification Ideas

### ISSUE THREAD

**Issue by garthvh** [meshtastic/Meshtastic-Apple#38]
### Deep Links

Create a structure using the new NavigationStack navigation features that allows for deep linking of assorted packet types

Potential links:
* [x] New node to the node details page
* [ ] Link Low Battery message to node details and the device telemetry details view
* [ ] Exchange Position Request Response Notification that opens node details with the map selected
* [x] Message notifications to the appropriate channel / dm and message
* [ ] Position / waypoint packets link directly to the mesh map

Examples 

https://swiftwithmajid.com/2022/06/21/mastering-navigationstack-in-swiftui-deep-linking/

https://tanaschita.com/20220502-quick-guide-on-local-notifications-for-ios/

### Watch Notifications

* [x] Figure out how to send notifications to watch
* [x] Make notifications open a tab
* [ ] Figure out how to make notifications that link to the message, not just the messages view

**Comment by Lolagatorade** [comment]
Yes much needed especially on phone and watch live activity buzzes the watch and phone continually. 


Also all notifications have Same priority. Discovering new nodes would be nice as a silent delivery whereas a direct DM or any message should have higher priority and actually buzz the watch or phone ect. 


Or have the option in settings to do so.
